<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElectorWidget</class>
 <widget class="QWidget" name="ElectorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>524</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Formulario Elector</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="gracias1">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>27</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>¡Gracias Por Votar!</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="gracias2">
       <property name="font">
        <font>
         <pointsize>27</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Tu participación hace la diferencia.</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="listaCandidatos">
     <property name="spacing">
      <number>9</number>
     </property>
     <property name="leftMargin">
      <number>32</number>
     </property>
     <property name="topMargin">
      <number>32</number>
     </property>
     <property name="rightMargin">
      <number>32</number>
     </property>
     <property name="bottomMargin">
      <number>32</number>
     </property>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
